src/libraries/System.Reflection.Emit/src/System/Reflection/Emit/GenericTypeParameterBuilderImpl.cs

```diff
@@ -83,7 +83,7 @@ protected override void SetInterfaceConstraintsCore(params Type[]? interfaceCons
         public override bool IsGenericType => false;
         public override bool IsGenericParameter => true;
         public override bool IsConstructedGenericType => false;
-        public override bool ContainsGenericParameters => _type.ContainsGenericParameters;
+        public override bool ContainsGenericParameters => false;
```

Comment:
Was the previous implementation incorrect? (where it forwarded to _type.ContainsGenericParameter)

Reply:
It was causing `StackOverflow` because of recursive call in https://github.com/dotnet/runtime/blob/b4491522e03c64a70fcac66dbe5a836e1a8dfffe/src/libraries/System.Private.CoreLib/src/System/Type.Helpers.cs#L58